temporal-sdk/src/main/java/io/temporal/client/UpdateWithStartWorkflowOperation.java

```diff
@@ -278,6 +278,8 @@ public static <R> Builder<R> newBuilder(String updateName, Class<R> resultClass,
 
   private Object[] updateArgs;
 
+  private Object[] workflowArgs;
```

Comment:
Can we add a comment here explaining where this is set and when? I think the current code isn't super obvious how this gets set since the AsyncInvocation is hard to follow.

Reply:
I agree, it took me a bit to re-figure it out myself.

Reply:
Yeah I think long term we should move this logic out of `UpdateWithStartWorkflowOperation.java` into its own class internal class that just deals with the the async invocation, but for now this is fine